identifiers/country-bg/national_level.csv

```diff
@@ -0,0 +1,33 @@
+id,name
+ocd-division/country:bg,Bulgaria
+ocd-division/country:bg/ed:1,Blagoevgrad
+ocd-division/country:bg/ed:2,Burgas
+ocd-division/country:bg/ed:3,Varna
+ocd-division/country:bg/ed:4,Veliko Tarnovo
+ocd-division/country:bg/ed:5,Vidin
+ocd-division/country:bg/ed:6,Vratsa
+ocd-division/country:bg/ed:7,Gabrovo
+ocd-division/country:bg/ed:8,Dobrich
+ocd-division/country:bg/ed:9,Kardzhali
+ocd-division/country:bg/ed:10,Kyustendil
+ocd-division/country:bg/ed:11,Lovech
+ocd-division/country:bg/ed:12,Montana
+ocd-division/country:bg/ed:13,Pazardzhik
+ocd-division/country:bg/ed:14,Pernik
+ocd-division/country:bg/ed:15,Pleven
+ocd-division/country:bg/ed:16,Plovdiv-city
+ocd-division/country:bg/ed:17,Plovdiv-province
+ocd-division/country:bg/ed:18,Razgrad
+ocd-division/country:bg/ed:19,Ruse
+ocd-division/country:bg/ed:20,Silistra
+ocd-division/country:bg/ed:21,Sliven
+ocd-division/country:bg/ed:22,Smolyan
+ocd-division/country:bg/ed:23,Sofia-city 23
+ocd-division/country:bg/ed:24,Sofia-city 24
+ocd-division/country:bg/ed:25,Sofia-city 25
+ocd-division/country:bg/ed:26,Sofia-province
```

Comment:
These seem like unlikely constituency names. Can we find any non-Wikipedia English election results for Bulgaria, to get another reference? (Wikipedia's quality isn't equally high for all countries.)

Reply:
http://electionresources.org/bg/assembly.php?election=2009 is only a little better Sofia 1 (23), Sofia 2 (24), Sofia 3 (25), Sofia District. Similarly: Plovdiv City and Plovdiv District. I'm sure there's something better.

Reply:
Good point. I updated the name using the list in the official website https://www.parliament.bg/en/MP.

Reply:
Awesome! Can you capitalize Grad and Oblast?

Reply:
Done.